packages/models/src/lib/implementation/schemas.ts

```diff
@@ -77,19 +85,22 @@ export function metaSchema(options?: {
   titleDescription?: string;
   descriptionDescription?: string;
   docsUrlDescription?: string;
+  scoreDescription?: string;
   description?: string;
 }) {
   const {
     descriptionDescription,
     titleDescription,
     docsUrlDescription,
+    scoreDescription,
     description,
   } = options || {};
   return z.object(
     {
       title: titleSchema(titleDescription),
       description: descriptionSchema(descriptionDescription),
       docsUrl: docsUrlSchema(docsUrlDescription),
+      score: scoreSchema(scoreDescription),
```

Comment:
This shouldn't be part of the meta schema. The group's `score` is not provided in the core config or plugin config, the audits haven't been run yet.

I see that each group already has a `score` in the `EnrichedScoredAuditGroup` used in `ScoredReport`, so it should already be available in `reportToMd`.

Reply:
Removed